### ISSUE TITLE
t.test.test_application.TestConvert fails without t.web

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11228]
|<img alt="marienz's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| marienz reported|
|-|-|
|Trac ID|trac#877|
|Type|defect|
|Created|2005-02-01 06:30:46Z|

<details><summary>Searchable metadata</summary>

```
trac-id__877 877
type__defect defect
reporter__marienz marienz
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1107239446000000 1107239446000000
changetime__1110644104000000 1110644104000000
version__ 
owner__ 
cc__radix cc__marienz
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="marienz's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|marienz commented|
|-|-|

```
#!html
<pre>
I get two failing tests here if I have only twisted-2.0.0a1 installed, not all
of twisted: testSimpleInternet fails because it can't import web.server and
testSimpleUnix fails because it can't import web.distrib. These tests should
either not need web, be part of web's tests, or be skipped if web is not installed.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
jml fixed this while he was fixing #876.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
